Source/Core/Core/FifoPlayer/FifoAnalyzer.cpp

```diff
@@ -219,6 +233,14 @@ u32 AnalyzeCommand(const u8* data, DecodeMode mode)
     break;
   }
 
+  case OpcodeDecoder::DOLPHIN_REPLAY_DATA:
```

Comment:
To be honest when initially coding this up, I wasn't expecting this to be hit, so I wasn't sure what to do when it was.